### PR TITLE
Proposed changes to standardized headings (One Metric Proof of Concept)

### PR DESCRIPTION
**Pull request by sgoggins**
If this standardization works for the **evolution** working group I will issue pull requests for the rest of the metrics to be released. 

Use Cases --> Objectives
Filter + Visualization placed under common heading of **Sample Filter and Visualization**
Test Cases (Examples) --> Examples
References --> Resources

### CODE REVIEW

File: metrics/Code_Changes.md [hunk 1]
```diff
@@ -101,7 +103,7 @@ Some useful visualizations are:
 These could be represented as bar charts, with time running in the X axis.
 Each bar would represent a code changes during a certain period (eg, a month).
 
-## Reference Implementation
+## Sample Implementation
```
Comment (jgbarah) [inline metrics/Code_Changes.md]: I would stick to "Reference", in the sense that if you're implementing, this is not just another implementation, that you can use as a sample, but a the reference implementation, that you should have into account, in the sense that the results of your implementation should be the same than this one. This is the whole idea of having reference implementations.

File: metrics/Code_Changes.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ Mandatory:
     those which do not touch files, and in some cases are not considered as
     reflecting a coding activity.
 
-## Use Cases
+## Objectives
```
Comment (jgbarah) [inline metrics/Code_Changes.md]: I'm not sure about "objectives". But I agree that captures the idea better than "use cases", and causes less confusion. So, if cannot find some other word, that's ok with me. Thanks!

**Comment by jgbarah** [comment]
Thanks for the idea, @sgoggins. I think it improves the template. Please, have a look at my two comments: I only have serious objections to one of the changes.

File: metrics/Code_Changes.md (hunk 1 above)
Comment (germonprez) [reply metrics/Code_Changes.md]: Thanks @jgbarah. The current template uses Sample. Now, I'm not saying that that's right :) I certainly see your point and moving Sample --> Reference in the other WGs wouldn't be difficult.

**Pull request merged**